ui/app/appointments/constants.js

```diff
@@ -24,13 +25,17 @@ Bahmni.Appointments.Constants = (function () {
         defaultCalendarEndTime: "19:00",
         defaultExpandServiceFilter: -1,
         collapseServiceFilter: 0,
+        defaultWeekDays: [0, 1, 2, 3, 4, 5, 6],
+        defaultWeekStartDay: 0,
+        defaultWeekStartDayName: 'Sunday',
         minDurationForAppointment: 30,
         appointmentStatusList: [ "Scheduled", "CheckedIn", "Completed", "Cancelled", "Missed" ],
         regexForTime: /^(?:(?:1[0-2]|0?[1-9]):[0-5]\d\s*[AaPp][Mm])?$/,
         privilegeManageAppointments: 'app:appointments:manageAppointmentsTab',
         privilegeForAdmin: 'app:appointments:adminTab',
         privilegeOwnAppointments: 'Manage Own Appointments',
         availableForAppointments: 'Available for appointments',
+        weekDays: {"Monday": 1, "Tuesday": 2, "Wednesday": 3, "Thursday": 4, "Friday": 5, "Saturday": 6, "Sunday": 7 },
```

Comment:
See above for related comment. If you use an objects for constants like this, you should leverage the above definition. Also, in this line - there is no "0", however for above "defaultWeekDays", it starts with "0".

Reply:
Used only one variable for getting the default value